.changeset/little-moons-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"@frontify/fondue": patch
+---
+
+bundle fondue-components
```

Comment:
```suggestion
build: bundle fondue-components
```

@noahwaldner @SamuelAlev should we consider something like committizen to automate this check on the changeset commit? Do we use conventional commits?